integr8ly/tutorial-web-app
# remove upgrade and maintenance window settings

## Motivation
https://issues.redhat.com/browse/APPDUX-423

## What
Remove the settings page for upgrade and maintenance window options

## Why
Restrict options so no upgrade / maintenance window outside of SRE coverage can be selected

## How
Remove the settings tab and homepage alert

## Verification Steps
1. Run the webapp locally: `yarn run start:dev`
2. Open the settings window: you should only see the tab to configure the solution patterns
